Stellar/soroban-starter/src/mappings/mappingHandlers.ts

```
@@ -1,118 +1,147 @@
+  logger.info(`VALUE TPE ${event.event.value.switch().name} `);
+
   // Create the new transfer entity
   const transfer = Transfer.create({
-    id: event.id,
-    ledger: event.ledger!.sequence,
-    date: new Date(event.ledgerClosedAt),
-    contract: event.contractId?.contractId().toString()!,
+    id: event.event.id,
+    ledger: event.event.ledger,
+    date: new Date(event.event.ledgerClosedAt),
+    contract: event.event.contractId?.contractId().toString()!,
```

Biome (2.3.13)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)